src/com/lftechnology/java_training/realworldproblem/CarImpl.java

```diff
@@ -0,0 +1,134 @@
+package com.lftechnology.java_training.realworldproblem;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This class implements various types of car Like {@link ElectricCar} and {@link PetrolCar} and its functionalities
```

Comment:
this comment misleads us as we might think this class is implementing the ElectricCar and PetrolCar interfaces